recipe/meta.yaml

```
@@ -1,40 +1,37 @@
 {% set name = "jupytext" %}
-{% set version = "1.15.2" %}
+{% set version = "1.16.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: c9976e24d834e991906c1de55af4b6d512d764f6372aabae45fc1ea72b589173
+  sha256: 68c7b68685e870e80e60fda8286fbd6269e9c74dc1df4316df6fe46eabc94c99
 
 build:
   number: 0
   entry_points:
     - jupytext = jupytext.cli:jupytext
-  skip: true # [py<36 or s390x]
-  script_env:
-    - BUILD_JUPYTERLAB_EXTENSION=1
+    - jupytext-config = jupytext_config.__main__:main
+  skip: true # [py<38 or s390x]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv
 
 requirements:
   host:
     - python
     - pip
-    - python
-    - nodejs 18.16.0
-    - jupyter-packaging
-    - jupyterlab >=3.0,<4.0
-    - setuptools
-    - wheel
+    - hatchling >=1.5.0
+    - hatch-jupyter-builder >=0.5
+    - jupyterlab >=4
   run:
     - python
     - nbformat
+    - mdit-py-plugins
+    - markdown-it-py >=1.0.0
```

```suggestion
    - markdown-it-py >=1.0
```
I don't think this change is that important, but just thought it be good to match upstream.

---

https://github.com/mwouts/jupytext/blob/v1.16.1/pyproject.toml#L36